client/src/components/Input.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import styled from "@emotion/styled";
+import PropTypes from "prop-types";
+
+const InputField = styled.input`
+  display: block;
+  height: 40px;
+  width: 80%;
+  background-color: ${({ theme }) => theme.colors.active};
+  border: none;
+  border-radius: 1px;
+  font-size: 18px;
+  margin: auto;
+  margin-top: 10px;
+  padding-left: 5px;
+`;
+
+export default function Input({ name }) {
+  return <InputField name={name} />;
```

Comment:
I don't see the use of the property `name` here, all you do is pass it along and then not use it :)
I think you could just go with the styled component here as well

Reply:
Thanks, i even don't know why i did this :D .